website/guides/workflow.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Workflow
+description: How does gql.tada and its LSP fit into your wofklow?
+---
+
+# Workflow
+
+There are some general workflows that are distinct locally (i.e. the LSP can take care of it)
+vs remotely. In this guide we'll look at how we can get local/remote to being consistent.
```

Comment:
> locally (i.e. the LSP can take care of it) [...] vs remotely

I don't think that's a complete picture here and might not be relevant immediately when clicking into this docs page.

Re. the terminology in general; I think “locally” includes the CLI, so I'd differentiate between:
- installation (excluded)
- opening a file and editing it (could be represented by a section linking to the GraphQLSP reference API doc)
- persisted (excluded)
- `generate output` if `tsc` is run but the editor is not
- saving the file and updating turbo
- remote: CI